homeassistant/components/octoprint/manifest.json

```diff
@@ -1,7 +1,14 @@
 {
   "domain": "octoprint",
-  "name": "OctoPrint",
+  "name": "Octoprint",
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/octoprint",
-  "after_dependencies": ["discovery"],
```

Comment:
https://github.com/home-assistant/core/blob/dev/homeassistant/components/discovery/__init__.py#L39

discovery should probably be updated as well

Reply:
Yes, it is available over zeroconf (and provides the unique ID as noted above as well, so it can automatically cancel is already configured).